python/templates/error.html

```diff
@@ -0,0 +1,6 @@
+<!DOCTYPE html>
+<html lang="en">
+<title>500 Internal Server Error</title>
```

Comment:
This webpage doesnt have body/head/meta charset/etc.

Reply:
yup -- but that's exactly how the flask 500 error renders for HTTP Exceptions. Happy to change it if you think it's necessary.

Reply:
Huh, okay. It does appear to pass W3C validation, so that's fine.